core/model/src/main/java/org/eclipse/sw360/antenna/api/Build.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.api;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+public class Build {
+    private final String directory;
+    private final String outputDirectory;
+    private final String sourceDirectory;
+    private final String testSourceDirectory;
+
+    public Build(String buildDirectory, String sourceDirectory) {
+        this.directory = buildDirectory;
+        this.outputDirectory = buildDirectory + File.separator + "classes";
+        this.sourceDirectory = Paths.get(sourceDirectory, "main", "java").toString();
```

Comment:
Would you keep still in the `api` package?

Reply:
if it is moved to another module, it also has to be moved to another package

Reply:
Oh you would move it to another module?

Reply:
This is Java specific and thus should be moved to some java related module.

This also depends on the actual project, since not only `main/java/` but also `main/kotlin/`, `main/scala/` or `main/thrift` are often present.